webpack.common.js

```diff
@@ -9,11 +9,24 @@ const loadAppConfig = () => {
   return {};
 };
 
-const createWebpackConfig = (
-  supportLegacyBrowsers
-) => {
+const babelOptions = {
+  presets: [
+    '@babel/preset-typescript',
+    '@babel/preset-react',
+    ['@babel/preset-env',
+    {
+      corejs: '3.6',
+      useBuiltIns: 'entry',
+    }]
+  ],
+  plugins: [
+    "@babel/plugin-proposal-class-properties",
+    "@babel/plugin-transform-runtime",
+    "react-hot-loader/babel"
```

Comment:
I believe we need to separate babel config and plugins for two different environments. 
The development mode do not needs `@babel/preset-env` and production do not require `react-hot-loader/babel`.